idd/Energy+.idd.in

```diff
@@ -24527,10 +24527,57 @@ ZoneEarthtube,
       \note  "C" in Equation
       \type real
       \default 0
- N18; \field Velocity Squared Term Flow Coefficient
+ N18, \field Velocity Squared Term Flow Coefficient
       \note  "D" in Equation
       \type real
       \default 0
+ A5,  \field Earth Tube Model Type
+      \type choice
+      \key Basic
+      \key Vertical
+      \default Basic
+ A6;  \field Earth Tube Model Parameters
+      \type object-list
+      \object-list EarthTubeParameterNames
+
+ZoneEarthtube:Parameters,
+      \memo Parameters that apply to the vertical model for an earth tube
```

Comment:
Should we do a min-fields here?

Reply:
Well, I guess that technically everything has a default so if the user just gave a name and ignored the rest it would just take all the defaults?  I wasn't sure so I just left that out.  I can put it in if you feel like this is important.